README.md

```diff
@@ -5,23 +5,24 @@
 [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/LiveIntent/live-connect.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/LiveIntent/live-connect/context:javascript)
-[![devDependencies Status](https://david-dm.org/LiveIntent/live-connect/dev-status.svg)](https://david-dm.org/LiveIntent/live-connect?type=dev)
-[![dependencies Status](https://david-dm.org/LiveIntent/live-connect/status.svg)](https://david-dm.org/LiveIntent/live-connect)
+[//]: # "[![devDependencies Status](https://david-dm.org/LiveIntent/live-connect/dev-status.svg)](https://david-dm.org/LiveIntent/live-connect?type=dev)"
+[//]: # "[![dependencies Status](https://david-dm.org/LiveIntent/live-connect/status.svg)](https://david-dm.org/LiveIntent/live-connect)"
 
-# Main concepts
-LiveConnect module is used to create, and or gather first party identifiers of your choosing, and sending that information to a defined endpoint which is responsible for gathering and processing that data.
-What LiveConnect provides is a simple interface to collect the identifiers from a page, and collect user interactions along with those identifiers.
-To see what kind of data is being sent, check [what is being sent](#what-is-being-sent).
+## Main concepts
+The LiveConnect module offers a convenient solution for generating and collecting first-party identifiers based on your preferences. It enables you to send this information to a designated endpoint, where it can be efficiently processed and analyzed. With LiveConnect, you gain a straightforward interface that facilitates the collection of identifiers from web pages, as well as capturing user interactions alongside these identifiers.
+If you're interested in reviewing the type of data being sent, please check [what is being sent](#what-is-being-sent) section of this documentation.
 
 ## Quick start
-- With [npm](https://www.npmjs.com/): `npm install`
-- Format the code according to the eslint file: `npm run fix-js`
-- Run unit tests: `npm run test:unit`
-- Run integration tests against a dockerized chrome: `npm run test:it:docker:chrome`
-- Run full integration tests on multiple browsers (requires properly set Browserstack credentials) : `npm run test:it:browserstack`
+To quickly get started with the LiveConnect module, perform the following steps via the command line interface.
+
+- Installation: Begin by installing the module using npm. Run the following command: `npm install`.
+- Code Formatting: Ensure your code adheres to the provided eslint file for consistent formatting. Use the command `npm run fix-js` to automatically format your code.
+- Unit Testing: Validate the functionality of your code by running the unit tests. Execute the command `npm run test:unit` to initiate the tests.
+- Integration Testing: Verify the integration of LiveConnect by running the integration tests against a dockerized Chrome browser. Use the command `npm run test:it:docker:chrome` to perform these tests.
```

Comment:
Before one can run integration tests locally, one needs to adjust some test domains (shell for Linux):
```
echo "127.0.0.1 bln.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 me.idex.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 schmoogle.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 framed.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 double-framed.test.liveintent.com" | sudo tee -a /etc/hosts
echo "127.0.0.1 baked.liveintent.com" | sudo tee -a /etc/hosts
```

Reply:
These domains are crucial if you run any of `npm run test:it:*`. This domain adjustment is provided in test section, but it's outdated there and incomplete